.travis.yml

```diff
@@ -65,6 +86,17 @@ before_install: |
   test "$TRAVIS_BRANCH" = master && TAGS="$TAGS latest" || true
   test -n "$TRAVIS_TAG"          && TAGS="$TAGS stable $TRAVIS_TAG" || true
   echo "Tags are $TAGS"
+  set -e
+  # The version encoded in pre-commit and built version must be in sync. If not,
+  # the pre-commit version was forgotten to be updated.
```

Comment:
This should be in `test/check_release` instead so it can be run before and not after a release